flash/data/process.py

```diff
@@ -128,7 +133,31 @@ def validating(self, val: bool) -> None:
             self._running_stage = None
 
 
-class Preprocess(Properties, Module):
+class AbstractPreprocess(ABC):
+
+    @abstractstaticmethod
+    def version() -> str:
+        """
+        Override to versioned to version your preprocess.
+        """
+        pass
```

Comment:
Why would the preprocess object have specific version? why not use `flash.__version__`?

Reply:
Because new way of processing the data might be implemented and the version should be dump.